db/migrate/20231101163013_create_jwt_denylists.rb

```diff
@@ -0,0 +1,10 @@
+class CreateJwtDenylists < ActiveRecord::Migration[7.0]
+  def change
+    create_table :jwt_denylists do |t|
+      t.string :jti, null: false
+      t.datetime :exp, null: false
+      t.timestamps
+    end
+    add_index :jwt_denylists, :jti
+  end
```

Comment:
I am not sure if this structure with the plural table for denylist will pose an issue later. The resources we used only have the table as singular as specified in the syllabus.